README.md

```diff
@@ -71,6 +71,19 @@ Then, render it in your views with the `component` helper (or its alias `c`).
 = c "button"
 ```
 
+Make sure to include javascript pack tag and stylesheet pack tag in your application layout file, for instance:
+
+```slim
+/ app/views/layouts/application.html.slim
+
+body=
+  = yield
+  = javascript_pack_tag "application"
+  = stylesheet_pack_tag "application"
```

Comment:
Please put the `stylesheet_pack_tag` in the `head` and add the doctype. See here for an example of slim syntax: http://slim-lang.com